src/commands/slash/mod/roles.ts

```diff
@@ -10,6 +10,8 @@ const ASSIGNABLE_ROLES = [
     '853016973804699689', // IRL
     '1333912863605002271', // politics timeout
     '1388337523012534342', // resident
+    '1339007768203366492', // chris fight
```

Comment:
role name may change
```suggestion
    '1339007768203366492', // thunderdome
```